.github/workflows/ci-docs.yml

```diff
@@ -0,0 +1,19 @@
+name: ci
```

Comment:
```suggestion
name: ci-docs
```

Reply:
unfortunately, the docs explicitly state that the names must be the same for the checks to match:

> Make sure that the name key and required job name in both the workflow files are the same. For more information, see "[Workflow syntax for GitHub Actions](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions)".

Reply:
Ah sorry, I did not RTFM